plugins/documents/lib/foodsoft_documents/engine.rb

```diff
@@ -11,5 +11,10 @@ def navigation(primary, context)
         sub_nav.items.insert(i, sub_nav.items.delete_at(-1))
       end
     end
+
+    def default_foodsoft_config(cfg)
+      cfg[:documents_allowed_extension] = 'gif|jpg|png|txt'
+      cfg[:use_documents] = true
```

Comment:
If we want to enable this by default, I suggest we change the default app config.

Reply:
it's just the same way as the other plugins do it, but i removed it for now. the only important part for me is, that the plugin is included in the `Gemfile`